activeactive/skills/skill-restore.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+
+red=$(tput setaf 1)
+green=$(tput setaf 2)
+yellow=$(tput setaf 3)
+default=$(tput sgr0)
+
+echo "$green Enter backup dir path:"
+read backupDir
+echo "$green Enter skill id:"
+read skillId
+
+uuidArray=(${skillId//-/ })
+uuid=${uuidArray[0]}
+
+echo -e "$green $(date) Backup started \n"
+
+# Validate dependency module
+# $1 - Name of the dependency module
+# $2 - Command to validate module
+function validate_dependency() {
+  eval $2
+  # Next statement is checking last command success
+  if [ $? -ne 0 ]; then
+    echo -e "$red $(date) Please install ******** $1 ***********  ... Exiting $default"
+    exit 1
+  fi
+}
+
+# Validate required modules exits in target setup
+function validate_setup() {
+  validate_dependency jq "jq --version"
+  echo -e "$(date) Successfully validated required dependencies"
+}
+
+
+function create_tenant_secret() {
+
+  echo -e "$green Starting port forwarding for ai-deployer to create tenant secret \n"
+
+  IDENTITY_SERVER_ENDPOINT=$(kubectl -n uipath get deployment ai-app-deployment -o json | jq -r '.spec.template.spec.containers[0].env[] | select(.name == "IDENTITY_SERVER_ENDPOINT").value')'/connect/token'
+  S2S_CLIENT_ID=$(kubectl -n uipath get secret identity-client-aifabric -o json | jq '.data."Aicenter.Recovery.S2S.ClientId"' | sed -e 's/^"//' -e 's/"$//' | base64 -d)
+  S2S_CLIENT_SECRET=$(kubectl -n uipath get secret identity-client-aifabric -o json | jq '.data."Aicenter.Recovery.S2S.ClientSecret"' | sed -e 's/^"//' -e 's/"$//' | base64 -d)
```

Comment:
Add validations to any input which we auto populate. 
Minimum validation should be empty check. IDENTITY_SERVER_ENDPOINT, S2S_CLIENT_ID, S2S_CLIENT_SECRET and ACCESS_TOKEN are all candidates.

Reply:
done